### DISCUSSION TITLE
Margin of Error

### DISCUSSION THREAD

**Discussion by rm17tink** [censusdis/censusdis#279] [Q&A]
I am using the ced.download() to pull in values which coincide with the estimates from the census website, is there a way to also pull in the margin of error for a given group?

**Comment by vengroff** [comment]
You can specify the ones you want manually, like this:

```python
import censusdis.data as ced
from censusdis.datasets import ACS5
from censusdis.states import NY, NJ, CT

ced.download(
    ACS5,
    2022,
    ['NAME', 'B19013_001E', 'B19013_001M'],

    state=[NY, NJ, CT]
)
```

But it would be nicer, especially when using `group=` or `leaves_of_group=` to have a flag `include_margin_of_error=True` that would get them automatically for all variables. Is that what you were thinking of?
